test/metabase/test/data/generic_sql.clj

```diff
@@ -130,6 +146,38 @@
             (qualify+quote-name driver database-name dest-table-name)
             (quot (pk-field-name driver)))))
 
+(defn standard-inline-column-comment-sql
+  "Generic inline COMMENT that driver can mixin if supported."
+  [_ field-comment]
+  (if field-comment
+    (format "COMMENT '%s'" field-comment)
+    (str "")))
+
+(defn standard-standalone-column-comment-sql
+  "Generic standalone COMMENT that driver can mixin if supported."
+  [driver {:keys [database-name]} {:keys [table-name]} {:keys [field-name field-comment]}]
+  (if field-comment
+    (format "COMMENT ON COLUMN %s IS '%s';"
+      (qualify+quote-name driver database-name table-name field-name)
+      field-comment)
+    (str "")))
```

Comment:
Same thing applies here -- `(str "")` is `""` -- but just returning nothing instead probably makes more sense